site/app/controllers/student/SubmissionController.php

```diff
@@ -113,7 +121,24 @@ private function deleteRequest(){
         }
         $this->core->getQueries()->deleteRegradeRequest($gradeable_id, $student_id);
     }
-
+    private function changeRequestStatus(){
+        $regrade_id = $_REQUEST['regrade_id'];
+        $gradeable_id = (isset($_REQUEST['gradeable_id'])) ? $_REQUEST['gradeable_id'] : null;
+        $student_id = (isset($_REQUEST['student_id'])) ? $_REQUEST['student_id'] : null;
+        $status = $_REQUEST['status'];
+        //TODO: set userViewedDate to null if the status is change to 1 to make the button green
+        if($status == 1){
+           // $this->core->getQueries()->getGradeable($gradeable_id, $student_id).setUserViewedDate(null);
+        }
+       // $gradeable_id = (isset($_REQUEST['gradeable_id'])) ? $_REQUEST['gradeable_id'] : null;
+      //  $gradeable=$this->core->getQueries()->getGradeable($gradeable_id);
```

Comment:
Lots of commented code here and an if-statement that does nothing... Were  you planning to add this to another PR instead?